website/site/content/docs/widgets/markdown.md

```diff
@@ -14,6 +14,7 @@ The markdown widget provides a full fledged text editor - which is based on [sla
 - **Data type:** markdown
 - **Options:**
   - `default`: accepts markdown content
+  - `buttons`: a list of string representing the buttons to display ('bold', 'italic', 'code', 'link', 'heading-one', 'heading-two', 'quote', 'code', 'bulleted-list', 'numbered-list'). If this options is missing, all the buttons are display.
```

Comment:
Update to:

> - `buttons`: an array of strings representing the formatting buttons to display, all buttons shown by default. Buttons include: "bold", "italic", "code", "link", "heading-one", "heading-two", "quote", "code-block", "bulleted-list", and "numbered-list".

While we happen to be using the types from our code, we shouldn't rely on this option aligning with type names. Note that I changed the second "code" to "code-block".